pulsar-broker/src/main/java/org/apache/pulsar/common/naming/NamespaceBundle.java

```diff
@@ -124,7 +124,7 @@ public String getBundleRange() {
         return bundleRange;
     }
 
-    private static String getKey(NamespaceName nsname, Range<Long> keyRange) {
+    public static String getKey(NamespaceName nsname, Range<Long> keyRange) {
         return String.format("%s/0x%08x_0x%08x", nsname, keyRange.lowerEndpoint(), keyRange.upperEndpoint());
```

Comment:
Please don't expose this method because of performance reasons. Instead, add an instance method  "getKey()" to NamespaceBundle so that the cached key can be retrieved. That seems to be your intention to get the key for the NamespaceBundle?

Reply:
The performance of String.format is bad and it causes a lot of allocations. It's fine to use it when there aren't a large number of invocations. That was optimized for NamespaceBundle in #9976.

Reply:
@lhotari  Thanks for your correction. PTAL again.